### PR TITLE
MULTIARCH-1971: IBM Z updates for OCP4.10

### PR DESCRIPTION
**Pull request by SNiemann15**
- OCP version for cherry-picking: enterprise-4.10

- Jira: https://issues.redhat.com/browse/MULTIARCH-1971
- BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2046520

- Related PR to remove CommonName workaround https://github.com/openshift/openshift-docs/pull/40187/
    - Preview [Mirroring the OpenShift Container Platform image repository](https://deploy-preview-41848--osdocs.netlify.app/openshift-enterprise/latest/updating/installing-update-service.html#update-service-mirror-release_update-service)
 
- Google doc: https://docs.google.com/document/d/11Wh9H_s1UBSxoY0tT3xqr0ZXNjusQcodzoUTDmdA3Nk/edit#

- Preview
   - [Installing a cluster with z/VM on IBM Z and LinuxONE](https://deploy-preview-41848--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html)
   - [Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network](https://deploy-preview-41848--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-restricted-networks-ibm-z.html)
   - [Installing a cluster with RHEL KVM on IBM Z and LinuxONE](https://deploy-preview-41848--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm.html)
   - [Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network](https://deploy-preview-41848--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.html)

- QE review IBM Z: Holger Wolf
- QE review for Multi-NIC: Muhammad Adeel (IBM Z) Micah Abbott (x86)

### CODE REVIEW

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -110,7 +110,7 @@ ip=10.10.10.3::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
 
 a|Optional: You can configure routes to additional networks by setting an `rd.route=` value.
 
-If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
+When you configure one or multiple networks, one default gateway is required. If the additional network gateway is different from the primary network gateway, the default gateway must be the primary network gateway.
```
Comment (SNiemann15) [inline modules/installation-user-infra-machines-static-network.adoc:113]: @miabbott I got feedback from the IBM Z QE @philc7777 that this change is also valid for x86. Can you please verify and ack. Thanks!

Comment (miabbott) [reply modules/installation-user-infra-machines-static-network.adoc:113]: Yeah, this makes sense to me.  :+1:

Comment (mtarsel) [reply modules/installation-user-infra-machines-static-network.adoc:113]: As the Power QE, LGTM

File: modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ Complete the following steps to create the machines in a fast-track installation
 
 .Prerequisites
 
-* At least 1 LPAR running {op-system-base} 8.3 with KVM, referred to as {op-system-base} KVM host in this procedure.
+* At least 1 LPAR running {op-system-base} 8.4 with KVM, referred to as {op-system-base} KVM host in this procedure.
```
Comment (nalhadef) [inline modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc:14]: Change "1" to "one."

File: modules/installation-requirements-user-infra-ibm-z-kvm.adoc [hunk 3]
```diff
@@ -151,7 +151,7 @@ Each cluster virtual machine must meet the following minimum requirements:
 [discrete]
 === Operating system requirements
 
-* For high availability, 2 or 3 LPARs running {op-system-base} 8.3 or later with KVM, which are managed via libvirt.
+* For high availability, 2 or 3 LPARs running {op-system-base} 8.4 or later with KVM, which are managed via libvirt.
```
Comment (nalhadef) [inline modules/installation-requirements-user-infra-ibm-z-kvm.adoc:154]: Change "2 or 3" to "two or three."

File: modules/installation-three-node-cluster.adoc [hunk 4]
```diff
@@ -31,7 +31,12 @@ endif::[]
 [id="installation-three-node-cluster_{context}"]
 = Configuring a three-node cluster
 
+ifndef::ibm-z,ibm-z-kvm[]
 You can optionally deploy zero compute machines in a bare metal cluster that consists of three control plane machines only. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for testing, development, and production.
+endif::ibm-z,ibm-z-kvm[]
+ifdef::ibm-z,ibm-z-kvm[]
+You can optionally deploy zero compute machines in a minimal three node cluster that consists of three control plane machines only. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for testing, development, and production.
```
Comment (nalhadef) [inline modules/installation-three-node-cluster.adoc:38]: Change "You can optionally deploy" to "Optionally, you can deploy."

File: modules/update-service-mirror-release.adoc [hunk 5]
```diff
@@ -21,12 +21,7 @@ endif::[]
 ifdef::openshift-origin[]
 * You have created a pull secret for your mirror repository.
 endif::[]
-* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
-+
-[source,terminal]
-----
-x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
-----
+* If you use self-signed certificates, you have specified a Subject Alternative Name in the certificates.
```
Comment (nalhadef) [inline modules/update-service-mirror-release.adoc:24]: Consider "You may use self-signed certificates if you have specified a Subject Alternative Name for the certificate."

**Comment by holgwolf** [comment]
lgtm

File: modules/installation-three-node-cluster.adoc (hunk 4 above)
Comment (SNiemann15) [reply modules/installation-three-node-cluster.adoc:38]: Also changing it in line 35 to be consistent.

File: modules/update-service-mirror-release.adoc (hunk 5 above)
Comment (SNiemann15) [reply modules/update-service-mirror-release.adoc:24]: To be consistent with the CommonName workaround PR mentioned in the description I will leave as is. But will bring up rewriting when I tackle merging the the two files as discussed [here](https://coreos.slack.com/archives/C85JYPHL3/p1644927941371739).

**Pull request merged**